spec/person_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../person'
+
+describe 'testing the person class' do
+  it 'initialize the person with age only' do
+    person = Person.new(18)
+    expect(person.age).to eq 18
+    expect(person.name).to eq 'unknown'
+    expect(person.parent_permission).to eq true
+  end
+
+  it 'should get correct name' do
+    person = Person.new(25, 'John Doe')
+
+    expect(person.correct_name).to eq 'John Doe'
+  end
```

Comment:
Awesome implementation for the person's class, kindly add test for the methods highlighted below as well as stated in the project requirement. ℹ️ 

<img width="368" alt="CAROL" src="https://github.com/hamidazim321/OOP_library/assets/103386183/3b786070-2b28-4665-be02-878cd6f5acdf">